app/src/main/AndroidManifest.xml

```diff
@@ -46,7 +46,13 @@
                 <data android:path="/redirect"/>
             </intent-filter>
         </activity>
+
         <meta-data android:name="firebase_analytics_collection_enabled" android:value="false" />
+        <meta-data android:name="firebase_crashlytics_collection_enabled" android:value="false" />
```

Comment:
do we want this to be turned off?

Reply:
Yes, this prevents analytics / crashes being sent before the user has opted in.

Reply:
It matches what id-check are doing and where the `enable` method on the FirebaseAnalyticsLogger calls both FirebaseAnalytics and FirebaseCrashlytics, ensures that they are synchronised at all times. I don't think there is any legal / GDPR reason for doing this with crashlytics (unlike the Analytics), more it is how the code expects it to be done.